packages/frontend/src/pages/HomePage.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { connect, useDispatch } from "react-redux";
+import { addMovie } from '../store/movie/actions';
+
+const CountClickComponent = () => {
```

Comment:
W jednym pliku twórz jedną classe, więc jak masz plik HomePage to staraj się tam mieć tylko classe o nazwie HomePage, a reszte które chcesz uzyć w tym pliku to importuj.

Reply:
Komponenty CountClickComponent oraz CountShow przeniosłam do osobny plików i teraz tylko importuje w HomePage.